src/index.html

```diff
@@ -18,5 +23,87 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <a
+        class="logo"
+        href="https://commons.wikimedia.org/wiki/File:LOGO_moyo_1.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="list">
+            <a
+              class="link link-blue is-active"
+              href="https://apple.com"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://samsung.com"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://sv.wikipedia.org/wiki/Smartmobil"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link link-blue"
+              href="https://en.wikipedia.org/wiki/Laptop"
+              data-qa="hover"
+            >
+              LaptopsS&Computers
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://en.wikipedia.org/wiki/Gadget"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://en.wikipedia.org/wiki/Tablet_computer"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://en.wikipedia.org/wiki/Photograph"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="list last-list">
```

Comment:
Avoid creating unique class names that represent the position or state like `last-list` unless it's necessary for styling purposes. If you need to target the last item for styling, consider using the `:last-child` or `:last-of-type` pseudo-class in your CSS.